playbooks/roles/plugins/foreman_docker/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+- name: 'Prepare hammer taxonomy options'
+  set_fact:
+    foreman_docker_hammer_taxonomy_params: "--organizations '{{ foreman_docker_organization }}' --locations '{{ foreman_docker_location }}'"
+  when: foreman_docker_use_taxonomies
+
+- name: 'Prepare blank hammer taxonomy options'
+  set_fact:
+    foreman_docker_hammer_taxonomy_params: ""
+  when: not foreman_docker_use_taxonomies
+
+- name: "Install foreman docker plugin"
+  yum:
+    name: 'tfm-rubygem-foreman_docker'
+    state: 'present'
+  tags:
+    - installation
+
+- name: "Install docker"
+  yum:
+    name: 'docker'
+    state: 'present'
+  tags:
+    - installation
+
+- name: 'Start and enable docker'
+  service:
+    name: docker
+    state: started
+  tags:
+    - installation
+
+- name: "Add usergroup docker"
+  group:
+    name: docker
+    state: present
+  tags:
+    - env_setup
+
+- name: "Add user foreman to group docker"
+  user:
+    name: foreman
+    groups: docker
+    append: yes
+  tags:
+    - env_setup
+
+- name: "Configure docker"
+  lineinfile:
+    dest: /etc/sysconfig/docker
+    regexp: '^OPTIONS='
+    line: "OPTIONS='--selinux-enabled -G docker'"
+  tags:
+    - env_setup
+
+- name: 'Restart Foreman to load the new compute provider type'
```

Comment:
I agree, if it was in the installer it would take care of reloading it. But as I mentioned in the previous comment, there's no option for installing foreman-docker.